src/WebJobs.Script/Workers/WorkerConcurrencyManager.cs

```diff
@@ -284,6 +296,24 @@ public void Dispose()
             Dispose(true);
         }
 
+        internal bool IsEnoughMemory(long hostProcessSize, IEnumerable<long> workerChannelSizes, long memoryLimit)
+        {
+            if (memoryLimit <= 0)
+            {
+                return true;
```

Comment:
I see `GetMemoryLimitBytes` can return -1. In that case, we return true here and will create a new worker process/scale. Is that expected?

![image](https://user-images.githubusercontent.com/144469/185008181-ed298e30-3267-43d6-8678-80f70c5c7437.png)

Reply:
Yes, it's expected as we do not want to block adding a new language worker when it's not possible to determinate memory limit